<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口助手</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#widget
{
	background-color: rgb(72, 144, 0);
}
#statusbar
{
	background-color: rgb(72, 144, 0);
}
#menubar
{
	background-color: rgb(72, 144, 0);
}
QComboBox
{
	 background-color:rgb(0, 170, 255);
	border-width: 0.5px;
    border-style: outset;
    border-color: rgb(90, 239, 255);
    color: rgb(255, 255, 255);
	padding: 3px;
}
QComboBox:pressed
{
	 background-color: rgb(144, 141, 150);
    border-style: inset;
}


QPushButton
{
	 background-color: rgb(0, 170, 255);
    border-style: outset;
    border-width: 0.5px;
    border-radius: 6px;
    border-color: rgb(90, 239, 255);
    color: rgb(255, 255, 255);
    font: 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed
{
	 background-color: rgb(144, 141, 150);
    border-style: inset;
}</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="4" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>第二步:开始烧录程序</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QTextBrowser" name="textEdit_ReceiveMsg"/>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBox_2">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_COMx">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pushButton_OpenSerial">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QComboBox" name="comboBox_3">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Refsh_Serial">
      <property name="text">
       <string>刷新串口</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#QpushButton
{
	background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>第一步发送要烧写的图片</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QComboBox" name="comboBox_Picture">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>256000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#label
{
	color:Black;
    font:14px;
}</string>
      </property>
      <property name="text">
       <string> 串口选择:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
